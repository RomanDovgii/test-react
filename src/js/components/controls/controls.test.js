import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Controls from "./controls";

it(
    `Should render correct controls`,
    () => {
      const renderer = new ShallowRenderer();

      const tree = renderer
      .render(
          <Controls/>
      );

      expect(tree).toMatchSnapshot();
    }
);
