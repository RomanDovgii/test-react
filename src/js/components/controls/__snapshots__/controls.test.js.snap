// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct controls 1`] = `
<section
  className="controls"
>
  <Memo(Connect(SearchPanel)) />
  <WithActiveAddUser />
</section>
`;
